nemo_text_processing/text_normalization/en/data/roman/README.md

```diff
@@ -0,0 +1,20 @@
+`female.tsv` - List of common female names. Copyright (c) January 1991 by Mark Kantrowitz, 4987 names, Version 1.3 (29-MAR-94)
```

Comment:
will this file type be added to pip package?

Reply:
yes, it's needed to build the roman graph